### DISCUSSION TITLE
nuget release

### DISCUSSION THREAD

**Discussion by gdoddsy** [BrighterCommand/Brighter#1440] [Q&A] [answered]
Hi,
I am trying to get started on a new project with Brighter, but I'm having trouble implementing the demos with the version in nuget (v8.1.1399). Am I doing this wrong, or has the nuget packages just not been updated yet?

**Comment by iancooper** [comment, marked as answer]
@gdoddsy We are about to release a new version, v9, and will need to update the demos. The GitHub feed (see the readme) will have a 'bleeding edge' release, but we expect to get an alpha out in the next week or so. The docs may be a bit behind that, and we welcome PRs when that happens

**Reply by gdoddsy** [reply]
@iancooper - thanks for the reply. It's not that the samples were too old, it's actually the opposite: they use features which don't exist in the nuget package. Things like Connections vs Subscriptions for the handlers. I worked out that I needed to find the samples at the time that the nuget package was made, and piecing that together worked out in the end. It almost felt like a nuget release was missed.

Once I got it going, it works well, it's just a steep learning curve. Thanks for the work that you've put into this project.

**Reply by iancooper** [reply]
That's just that V9, which the samples are for, isn't out on the public NuGet server yet. Next week or so. But you can always get the a nuget package from our GitHub package feed that represents the last good build. Caveat Emptor there though, as it's work in flight.

We will be using MinVer to tag releases so that you will find it easy to see the source that matches a release in future

**Reply by iancooper** [reply]
We develop on 'master' so it may be ahead of any NuGet package
